mdslib/testing/mdslib_ftest.f

```diff
@@ -27,9 +27,9 @@ program mdslib_test
       enddo
 
       dsc = descr(IDTYPE_FLOAT,result,0,0)
```

Comment:
I think we should add another test at the bottom, instead of replacing the last test

Reply:
@joshStillerman I would still put this test back as it was, and add another test that tests the `size=1`